src/tripod.inc.php

```diff
@@ -1,19 +1,42 @@
 <?php
-if(!defined('TRIPOD_DIR')) define('TRIPOD_DIR', dirname(__FILE__) . '/' );
 
-if (version_compare(phpversion(), '5.3.0', '<')) {
-    // php version isn't high enough for ARC, use legacy <5.3.0 version
-    if(!defined('ARC_DIR')) define('ARC_DIR', TRIPOD_DIR. '../lib/arc/' );
-}
-else
-{
-    // use latest brought in via composer
-    if(!defined('ARC_DIR')) define('ARC_DIR', TRIPOD_DIR. '../vendor/semsol/arc2/' );
-}
+//todo: this file is mis-named. It has mongo specifics
```

Comment:
Again one for @robotrobot, since I'm not sure I 100% understand how it should work differently.  Everything that is mongo-specific (I think) would be addressed sufficiently if Tripod was PSR-compliant.